contribs/drt/src/main/java/org/matsim/contrib/drt/routing/DrtRouteCreator.java

```diff
@@ -65,17 +65,30 @@ static double getMaxTravelTime(DrtConfigGroup drtCfg, double unsharedRideTime) {
 				drtCfg.maxTravelTimeAlpha * unsharedRideTime + drtCfg.maxTravelTimeBeta);
 	}
 
+	/**
+	 * Calculates the maximum ride time defined as: drtCfg.maxDetourAlpha * unsharedRideTime + drtCfg.maxDetourBeta
+	 *
+	 * @param drtCfg
+	 * @param unsharedRideTime ride time of the direct (shortest-time) route
+	 * @return maximum ride time
+	 */
+	static double getMaxRideTime(DrtConfigGroup drtCfg, double unsharedRideTime) {
+		return Math.min(unsharedRideTime + drtCfg.maxAbsoluteDetour, drtCfg.maxDetourAlpha * unsharedRideTime + drtCfg.maxDetourBeta);
```

Comment:
I am not sure if we should use `unsharedRideTime + drtCfg.maxAbsoluteDetour` in both `getMaxTravelTime()` and `getMaxRideTime()`.

Imposing this constraint on `maxTravelTime` implies it is also imposed on `maxRideTime`, because `maxTravelTime >= maxRideTime`.